src/main/java/graphql/execution2/DefaultExecutionStrategy.java

```diff
@@ -0,0 +1,112 @@
+package graphql.execution2;
```

Comment:
Why the `execution2` path.  Why not called leave it in `graphql.execution` and rename the class.

is this intended to be the packahe on going.  `graphql.xxx2` smells to me